src/types/production_dependency_config_options.go

```diff
@@ -2,7 +2,12 @@ package types
 
 // ProductionDependencyConfigOptions represents the configuration of a development dependency
 type ProductionDependencyConfigOptions struct {
-	//TODO: populate with RDS fields
+	AllocatedStorage string `yaml:"allocated-storage,omitempty"`
+	InstanceClass    string `yaml:"instance-class,omitempty"`
+	DbName           string `yaml:"db-name,omitempty"`
+	Username         string `yaml:",omitempty"`
+	PasswordEnvVar   string `yaml:"password-env-var,omitempty"` //TODO: verify this is set via exo configure
+	StorageType      string `yaml:"storage-type,omitempty"`
```

Comment:
Weren't we going to group this in an RDS config?

Reply:
Right, I'll add that in